include/grpcpp/ext/csm_observability.h

```diff
@@ -100,7 +98,8 @@ class OpenTelemetryPluginOption;
 ///     .SetMeterProvider(provider)
 ///     .AddPluginOption(MakeCsmOpenTelemetryPluginOption())
 ///     .BuildAndRegisterGlobal();
-std::unique_ptr<OpenTelemetryPluginOption> MakeCsmOpenTelemetryPluginOption();
+std::unique_ptr<grpc::OpenTelemetryPluginOption>
+MakeCsmOpenTelemetryPluginOption();
```

Comment:
Shouldn't this move out of the `experimental` namespace as well?  Or do we want to do that as a separate step?